cmd/kubeapps-apis/docs/kubeapps-apis.swagger.json

```diff
@@ -4582,6 +4574,14 @@
           "description": "An example of default values used during package templating that can serve\nas documentation or a starting point for user customization.",
           "title": "Available package default values"
         },
+        "customDefaultValues": {
```

Comment:
this is just a suggestion, but maybe we can use a more neutral word? The description is using the term extra for example, another term could be "additional" or "alternate"?

Reply:
I'd go for "additional" (as they are layered on top) rather than "alternate" (as they are not instead of), but yep, "additional" is good for me (better than "custom"). I'll update to that. Thanks.